src/views/posts-index.js

```diff
@@ -0,0 +1,44 @@
+var RunList = Backbone.View.extend({
```

Comment:
This represents an individual item not the `RunList` so you will want to name it `RunView` or something similar so that you will keep things straight when you come back to the code.